lib/util/state.js

```diff
@@ -827,49 +825,33 @@ export function getUiUrlParams(state) {
   return uiParams
 }
 
-// Set default title to the original document title (on load) set in index.html
-const DEFAULT_TITLE = document.title
-
-export function getTitle(state, intl) {
-  // Override title can optionally be provided in config.yml
-  const { config, ui } = state.otp
-  const { localUser, loggedInUser } = state.user
-  const user = loggedInUser || localUser
-  const title = config.title || DEFAULT_TITLE
-  const { mainPanelContent, viewedRoute, viewedStop } = ui
-  let status
-  switch (mainPanelContent) {
-    case MainPanelContent.ROUTE_VIEWER:
-    case MainPanelContent.PATTERN_VIEWER:
-      status = intl.formatMessage(
-        { id: 'util.state.titleBarRouteId' },
-        { routeId: viewedRoute?.routeId }
-      )
-      break
-    case MainPanelContent.STOP_VIEWER:
-      status = intl.formatMessage(
-        { id: 'util.state.titleBarStopId' },
-        {
-          stopId:
-            viewedStop && coreUtils.itinerary.getDisplayedStopId(viewedStop)
-        }
-      )
-      break
-    default:
-      // eslint-disable-next-line no-case-declarations
-      const activeSearch = getActiveSearch(state)
-      if (activeSearch) {
-        status = summarizeQuery(activeSearch.query, intl, user.savedLocations)
-      }
-      break
-  }
+export function getRouteOperator(routeObject, transitOperators) {
+  return (
+    coreUtils.route.getTransitOperatorFromOtpRoute(
+      routeObject,
+      transitOperators
+    ) || {}
+  )
+}
 
-  if (status) {
-    return intl.formatMessage(
-      { id: 'util.state.titleBarWithStatus' },
-      { status, title }
-    )
-  }
+/** Helper method to get the user-friendly operator name (fallback on GTFS agency name). */
+export function getOperatorName(operator, agencyName) {
+  return operator?.name || agencyName
+}
```

Comment:
Is this just to better handle all the possible `agency.name || agencyName` etc? This feels slightly more complicated than doing an inline check to me

Reply:
Sure, addressed in https://github.com/opentripplanner/otp-react-redux/pull/819/commits/fa09f2381a6089fcba746e2748104bec1408cb23.